src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1492,7 +1493,7 @@ static bool addTriggeredSubscriptions_noCache
         (
           throttling,
           lastNotification,
-          sub.hasField(CSUB_FORMAT) ? stringToFormat(getStringField(sub, CSUB_FORMAT)) : XML,
+          sub.hasField(CSUB_FORMAT) ? stringToFormat(getStringField(sub, CSUB_FORMAT)) : JSON,
```

Comment:
I thought in removing format suff in mongoBackend, but I realized that it could be useful to implement the different notifications formats introduced in PR https://github.com/telefonicaid/fiware-orion/pull/1761. The current "JSON" will stand for legacy format, and we could add new ones for "keyValues-array", "keyValues-keymap", etc.

NTC (just a remark)